server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/GitWithAuth5.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.centraldogma.server.internal.mirror;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.eclipse.jgit.api.FetchCommand;
+import org.eclipse.jgit.api.TransportCommand;
+import org.eclipse.jgit.transport.JschConfigSessionFactory;
+import org.eclipse.jgit.transport.OpenSshConfig.Host;
+import org.eclipse.jgit.transport.SshTransport;
+
+import com.jcraft.jsch.Session;
+
+import com.linecorp.centraldogma.server.MirrorException;
+import com.linecorp.centraldogma.server.internal.mirror.credential.PasswordMirrorCredential;
+import com.linecorp.centraldogma.server.internal.mirror.credential.PublicKeyMirrorCredential;
+
+public class GitWithAuth5 extends GitWithAuth {
+    GitWithAuth5(GitMirror mirror, File repoDir) throws IOException {
+        super(mirror, repoDir);
+    }
+
+    @Override
+    FetchCommand fetch(int depth) {
+        throw new UnsupportedOperationException();
```

Comment:
Shouldn't we still return a `FetchCommand` without calling `setDepth()`?

Reply:
It's OK, updated!